src/base/ibstream.rs

```diff
@@ -11,7 +11,7 @@ pub trait IBStream: IUnknown {
         -> tresult;
     unsafe fn write(
         &self,
-        buffer: *mut c_void,
+        buffer: *const c_void,
```

Comment:
this needs to be `mut` I think?

Reply:
This one is a matter of taste. First of all, the naming of the IBStream methods are kinda backwards: write is for reading from the buffer (so *const would make sense), read is for writing to the buffer. You can see that for example in the again example in the SDK (and also in the again example in vst3-sys).